terasoluna-tourreservation-web/src/main/resources/META-INF/spring/application.properties

```diff
@@ -2,3 +2,6 @@
 customer.initialBirthYear=1980
 customer.initialBirthMonth=1
 customer.initialBirthDay=1
+
+reservation.report.pdf.name=reservationReport.pdf
```

Comment:
- [ ] Good to fix file name extension ( `.pdf` ). Because extension (file type) is determined by View.

Reply:
3b4cd87dc7b6a6f04c3bb7ff7e58311580d37bdf

I change to determine extension in View class in above commit .

Reply:
I think extension should be constant field value.

Reply:
7f2ec019427c98f9961c7280a3636c1298e43873

I changed extension to constant field in above commit .

Reply:
The constant field should contain `.`.

Reply:
950acf667499d3eb8d8f287e46ab972de56bc089

I changed to cantain "." in acnstant field in above commit .